core/src/main/scala/better/files/UnicodeCharset.scala

```diff
@@ -12,11 +12,12 @@ import scala.collection.JavaConverters._
   * @param writeByteOrderMarkers If set, write BOMs while encoding
   */
 class UnicodeCharset(underlyingCharset: Charset, writeByteOrderMarkers: Boolean)
-    extends Charset(underlyingCharset.name(), underlyingCharset.aliases().asScala.toArray) {
+    extends Charset(underlyingCharset.name(),
+                    underlyingCharset.aliases().asScala.toArray.map(_.asInstanceOf[String | Null])) {
   override def newDecoder() = new UnicodeDecoder(underlyingCharset)
   override def newEncoder() =
     if (writeByteOrderMarkers) new BomEncoder(underlyingCharset) else underlyingCharset.newEncoder()
-  override def contains(cs: Charset) = underlyingCharset.contains(cs)
+  override def contains(cs: Charset | Null) = underlyingCharset.contains(cs)
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.